puppet/modules/quickstack/manifests/dellsc/volume.pp

```diff
@@ -1,4 +1,4 @@
-define quickstack::dell_sc_iscsi::volume (
+define quickstack::dellsc::volume (
```

Comment:
perhaps then this should be changed to be:
(directory structure) manifests/dell/sc_iscsi/volume.pp
resulting in a define with the name:
quickstack::dell::sc_iscsi::volume
Would that be clearer and allow you to fold the eql code into the quickstack::dell namespace as desired?